etc/profile-a-l/clion.profile

```diff
@@ -5,13 +5,13 @@ include clion.local
 # Persistent global definitions
 include globals.local
 
-nodeny  ${HOME}/.CLion*
+nodeny  ${HOME}/.config/JetBrains/CLion*
+nodeny  ${HOME}/.cache/JetBrains/CLion*
 nodeny  ${HOME}/.config/git
 nodeny  ${HOME}/.gitconfig
 nodeny  ${HOME}/.git-credentials
 nodeny  ${HOME}/.java
 nodeny  ${HOME}/.local/share/JetBrains
-nodeny  ${HOME}/.tooling
```

Comment:
Reason? (IDK what this is)

Reply:
idk what this is

Reply:
You removed because you don't know what it is ???!

Reply:
```console
$ grep "\.tooling" etc/**.profile
etc/profile-a-l/android-studio.profile:nodeny  ${HOME}/.tooling
etc/profile-a-l/aosp.profile:nodeny  ${HOME}/.tooling
etc/profile-a-l/clion.profile:nodeny  ${HOME}/.tooling
etc/profile-a-l/idea.sh.profile:nodeny  ${HOME}/.tooling
etc/profile-m-z/webstorm.profile:nodeny  ${HOME}/.tooling
```

Maybe a allow-common-devel.inc candidate.

Reply:
I guess it has something to do with java.

Reply:
But for what kinda reason `${HOME}/.tooling` exist?

Reply:
Can you prove that _nothing_ is broken if you deny access to it?

Reply:
What feature is depends on that folder?

Reply:
FYI: https://github.com/eclipse/eclipse.jdt.ls/pull/1655

Reply:
> But for what kinda reason ${HOME}/.tooling exist?
> What feature is depends on that folder?

 **IDK what this is** but removing it because we don't know what it is is wrong. Removing a `nodeny` command is always wrong unless you can prove (not show) that it is not used.